docs/schemas/Library-Minimal-Metadata/Library Minimal Metadata.csv

```diff
@@ -0,0 +1,52 @@
+library avg size bp,f851eba9-56d1-4472-9d0c-d7f8bc33000a,,,"Average size of the library in terms of base pairs.  This is used to calculate the molarity before pooling and sequencing.
+",,,TRUE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library method,7b60d59e-fdd7-4b27-a2d4-cae9b69103a6,library chemistry method,,"Standardized nomenclature to describe the library method used.  This specifies the alignment method required for the library.  For example, 10xV3.1 (for RNASeq single assay), 10xMult-GEX (for RNASeq multiome assay), and 10xMult-ATAC (for ATACSeq multiome assay)",,,TRUE,,ValueSet,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Library concentration nm,90805b3f-f380-4f23-b159-e7eaa0c8f052,,,Concentration of library in terms of nM (nMol/L).  Number of molecules is needed for accurate pooling of the libraries and for generating the number of target reads/cell in sequencing.,,,FALSE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library creation date,9c2f575d-1b64-451d-894f-656861afe07a,library construction date,,Date of library construction,,,FALSE,,Date,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Library input ng,e4d31d97-722d-4771-a0e4-e6062190f2c1,,,Amount of cDNA going into library construction in nanograms.,,,TRUE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library label,f717e254-3630-4342-be7b-4d56376e7afe,library name,,"Name of a library, which is a collection of fragmented and barcode-indexed DNA molecules for sequencing.  An index or barcode is typically introduced to enable identification of library origin to allow libraries to be pooled together for sequencing.",,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library prep pass-fail,6817ede2-7ead-402d-9dbc-131aca627c6c,library prep pass-fail result,,Pass or Fail result based on qualitative assessment of library yield and size.,,,TRUE,,ValueSet,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+Library prep set,b124ffa9-9134-4a61-a30d-bb191b2fc7fa,,,"Library set, containing multiple library_names that were processed at the same time.",,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library quantification fmol,4c09ada7-c116-48bc-8fb1-0dcf5c4b939a,,,Amount of library generated in terms of femtomoles,,,TRUE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+library quantification ng,318b2d3a-dae7-4c63-bfbb-93862b92f63e,,,"Amount of library generated in terms of nanograms
+",,,FALSE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```

Comment:
library quantification ng: expected data type -> float
data example provided in green table: 1376.612572
```suggestion
",,,FALSE,,Float,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```